exchanges/kucoin/kucoin_types.go

```diff
@@ -956,19 +956,6 @@ type WsPriceIndicator struct {
 	Value       float64              `json:"value"`
 }
 
```

Comment:
remove error `errSubscriptionPairRequired` top of file please.

Reply:
Fixed b090d967